.github/CHANGELOG.md

```diff
@@ -121,6 +121,9 @@
   an owner of the pruned tensor and its constituent observables, but leaves the
   original tensor in the queue without an owner.
 
+* The `qml.ResetError` is now supported for `default.mixed` device. 
```

Comment:
Nice! :slightly_smiling_face:  Could you add yourself to the list of contributors too on line 137?

Reply:
Yep! I have just added my name.